solang-parser/src/lexer.rs

```diff
@@ -178,6 +178,8 @@ pub enum Token<'input> {
     Case,
     Default,
     YulArrow,
+
+    Annotation(&'input str),
     At,
```

Comment:
`Token::At` is now unused, right? I don't think there is any reason to keep it.

Reply:
Fixed!